### DISCUSSION TITLE
Miktex for Debian 11 Bullseye

### DISCUSSION THREAD

**Discussion by ghost** [MiKTeX/miktex#919] [Q&A]
# Background
I recently upgraded to the Debian 11 **WSL2** image from the Windows store. However, when I wanted to [install](https://miktex.org/howto/install-miktex-unx) `miktex` on my system, instructions for the latest Debian release have not yet been updated.

# Question
When will there be `miktex` support for Debian 11?

**Comment by Paebbels** [comment]
I'm currently using Buster packages with my Bullseye. It works so far.

But yes official support would be welcome, especially because Bullseye exists since years.
